lib/elves.c

```diff
@@ -488,6 +488,7 @@ fde_read_address(const uint8_t *p, unsigned len)
         uint32_t n4;
         uint64_t n8;
     } u;
+    u.n8 = 0;
```

Comment:
How does this work? There is never another assignment to u.n8, but neither to u.n4 (which you did not initializes too).

Reply:
u is C union, can represent any of its members and its memory is of size of its biggest member.
Warning was, that the function returns possibly uninitialized value, which is true since the value of union is initialized depending on its len argument.
n8 represents all 8 bytes of the union, so initializing n8 makes whole union set to zero and I think it is also shortest and nicest possible solution to initialize the biggest element of the union